ui/src/store/testdata/rating_resource_13.json

```diff
@@ -0,0 +1 @@
+{ "rating": 2 }
```

Comment:
Since it is a single line data should we add a new file for this or can we just use a constant ?? :thinking:  wdyt ?